taskcluster/script.py

```diff
@@ -53,6 +53,7 @@ def get_device_type(device):
 
 
 def enable_charging(device, device_type):
+    print("script.py: enabling charging...")
```

Comment:
Might as well print the device and device_type.

Reply:
Done.